overview.md

```diff
@@ -43,130 +41,180 @@ steps:
 
-```YAML
-pool: myPool
+```
+pool:
+  vmImage: ubuntu-latest
 steps:
+  - task: InstallMATLAB@1
+    inputs:
+      release: R2023b
   - task: RunMATLABCommand@1
     inputs:
       command: myscript
-``` 
+```
 
-### Specify MATLAB in Pipeline
-When you use the **Run MATLAB Build**, **Run MATLAB Tests**, or **Run MATLAB Command** task in your pipeline, the self-hosted agent uses the topmost MATLAB version on the system path. The pipeline fails if the agent cannot find any version of MATLAB on the path.
+### Specify MATLAB Release on Self-Hosted Agent
+When you use the **Run MATLAB Build**, **Run MATLAB Tests**, or **Run MATLAB Command** task in your pipeline, the agent uses the topmost MATLAB release on the system path. The task fails if the agent cannot find any release of MATLAB on the path.
 
-You can prepend your preferred version of MATLAB to the `PATH` environment variable of the agent. For example, prepend MATLAB R2023b to the path and use it to run your script.
+In R2021a and later, you can use the **Install MATLAB** task to prepend your preferred release of MATLAB to the `PATH` system environment variable of the agent. You can also add your preferred release to the path without using the **Install MATLAB** task. For example, prepend MATLAB R2020b, which the **Install MATLAB** task does not support, to the path and use it to run your script. The step depends on your operating system and MATLAB root folder.
 
-```YAML
+```
 pool: myPool
 steps:
-  - powershell: Write-Host '##vso[task.prependpath]C:\Program Files\MATLAB\R2023b\bin'  # Windows agent
-# - bash: echo '##vso[task.prependpath]/usr/local/MATLAB/R2023b/bin'  # Linux agent
+  - powershell: Write-Host '##vso[task.prependpath]C:\Program Files\MATLAB\R2020b\bin'  # Windows agent
+# - bash: echo '##vso[task.prependpath]/usr/local/MATLAB/R2020b/bin'  # Linux agent
   - task: RunMATLABCommand@1
     inputs:
       command: myscript
 ```
 
-### Use MATLAB on Microsoft-Hosted Agent
-Before you run MATLAB code or Simulink models on a Microsoft-hosted agent, first use the [Install MATLAB](#install-matlab) task. The task installs your specified MATLAB release (R2021a or later) on a Linux&reg; virtual machine. If you do not specify a release, the task installs the latest release of MATLAB.
+### Install Transformation Product on Microsoft-Hosted Agent
+To use a transformation product, such as MATLAB Coder&trade; and MATLAB Compiler&trade;, on a Microsoft-hosted agent, you need a MATLAB batch licensing [token](https://github.com/mathworks-ref-arch/matlab-dockerfile/blob/main/alternates/non-interactive/MATLAB-BATCH.md#matlab-batch-licensing-token). Batch licensing tokens are strings that enable MATLAB to start in noninteractive environments. You can request a token by contacting MathWorks&reg; at [batch-tokens@mathworks.com](mailto:batch-tokens@mathworks.com).
```

Comment:
It looks like in our [program offering guide](https://www.mathworks.com/help/pdf_doc/offering/offering.pdf) these are called "Transformation Programs". I'm not sure most customers know what Transformation Program means, but it is basically anything with "coder" or "compiler" in the name plus a few other products.

Reply:
Thanks for the link. I can see the list of products on page 25.

Unfortunately, we don't use "transformation program" or "transformation product" in MathWorks documentation. We came up with "transformation product" when first documenting our cloud CI offerings.

We can definitely use "transformation program" instead. Any suggestions, @mcafaro?

Reply:
I agree "transformation product" makes more sense to me because our parameter is called `products`. I also think the first sentence gives some context about what it means.

Reply:
Good point about the `products` argument. However, it's still unfortunate that "transformation product or program" doesn't result in any useful web search results.

We can also say "products that transform MATLAB code, ...", but not sure how better it will be.

Reply:
Either works for me. AFAIK, the offering guide is the only place with this list and getting to the PDF in our doc requires logging in with a MathWorks account. Do you think it is worth while/possible/allowed for us to link to the offering guide? If so, I think we should use the terminology that aligns best with it.